02-User-Profile/README.md

```diff
@@ -1,28 +1,32 @@
 # Auth0 React User Profile
 
-This sample demonstrates how to get a user's profile using Auth0 in a React application. The sample uses create-react-app.
+This sample demonstrates how to get a user's profile using Auth0 in a React application.
 
 ## Getting Started
 
-If you haven't already done so, [sign up](https://auth0.com) for your free Auth0 account and create a new client in the [dashboard](https://manage.auth0.com). Find the **domain** and **client ID** from the settings area and add the URL for your application to the **Allowed Callback URLs** box. If you are using the server provided by the create-react-app, that URL is `http://localhost:3000/callback`.
+If you haven't already done so, [sign up](https://auth0.com) for your free Auth0 account and create a new client in the [dashboard](https://manage.auth0.com). Find the **domain** and **client ID** from the settings area and add the URL for your application to the **Allowed Callback URLs** box. The default URL is `http://localhost:3000/callback`. Also configure **Allowed Web Origins** to the default application URL `http://localhost:3000`.
 
-Clone the repo or download it from the React quickstart page in Auth0's documentation. Install create-react-app globally and the dependencies for the app.
+Clone the repo or download it from the React quickstart page in Auth0's documentation.
+
+Open the first demo.
```

Comment:
Should we be referring the reader to open the first demo, even though they're currently reading the readme for demo 2? It would make more sense to me if we said 'Open the demo' and then asked them to `cd` into `02-User-Profile` (see command below)

https://github.com/auth0-samples/auth0-react-samples/blob/15e800fe0e6262800b96f74723d9136e0dea0802/02-User-Profile/README.md#L11-L14

Reply:
Ooops :D Copypasta.